graphql/resolvers.go

```diff
@@ -140,6 +142,25 @@ func (r *taskResolver) ReliesOn(ctx context.Context, at *restModel.APITask) ([]*
 	return dependencies, nil
 }
 
+func (r *taskResolver) Ami(ctx context.Context, at *restModel.APITask) (*string, error) {
+	d, err := distro.FindByID(*at.DistroId)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error while searching for distro: '%s", *at.DistroId))
+	}
+	if d == nil {
+		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("Could not find distro with id: '%s", *at.DistroId))
```

Comment:
the error is not included in this error

Reply:
There is no error to propagate here, `distro.FindById` will return nil if it cant find a distro with no other error.